src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractServiceImpl.java

```diff
@@ -160,7 +162,7 @@ private PdpFeedFileBaseEntry buildPdpFeedFileBaseEntry(ConcurStandardAccountingE
         int totalReimbursementLineCount = 0;
         KualiDecimal totalReimbursementDollarAmount = KualiDecimal.ZERO;
         for (ConcurStandardAccountingExtractDetailLine line : concurStandardAccountingExtractFile.getConcurStandardAccountingExtractDetailLines()) {
-            if (StringUtils.equalsIgnoreCase(line.getPaymentCode(), ConcurConstants.PAYMENT_CODE_CASH)) {
+            if (shouldProcessSAELineToPDP(line)) {
                 totalReimbursementLineCount++;
                 totalReimbursementDollarAmount = totalReimbursementDollarAmount.add(line.getJournalAmount());
```

Comment:
Since the conditions for entering this code block have changed, you may also need to update the code that calculates the total reimbursement count and amount. I don't think cash advances or corp-card personal expenses should be included in the total for that item in the report.